application/console/components/AwsStartupAction.php

```diff
@@ -7,11 +7,13 @@
 use common\components\Appbuilder_logger;
 use common\components\CodeBuild;
 use common\components\IAmWrapper;
+use common\components\S3;
 
 use common\helpers\Utils;
 
 class AwsStartupAction extends ActionCommon
 {
+    private $s3;
```

Comment:
Why is this a private member when it is only used in copyDefaultProjectFolder?

Reply:
I'll fix this.  I think I was leaving it thinking towards merging it but I'll change it

Reply:
I'll fix this.  I think I was leaving it thinking towards merging it but I'll change it